docs/flow/modin/experimental/pandas.rst

```diff
@@ -12,5 +12,4 @@ Experimental API Reference
 .. autofunction:: read_sql
 .. autofunction:: read_csv_glob
 .. autofunction:: read_pickle_distributed
-.. autoclass:: DataFrame
-   :members: to_pickle_distributed
```

Comment:
Thus, the library generated 2 targets with the same `DataFrame` name, which resulted in a warning. (`...\projects\modin\modin\pandas\base.py:docstring of modin.pandas.base.BasePandasDataset.var:: WARNING: more than one target found for cross-reference 'DataFrame': modin.experimental.pandas.DataFrame, modin.pandas.dataframe.DataFrame`)